watertap/property_models/tests/test_cryst_prop_pack.py

```diff
@@ -0,0 +1,804 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+import pytest
+import watertap.property_models.cryst_prop_pack as props
+from pyomo.environ import ConcreteModel, SolverFactory, TerminationCondition
+from idaes.core import FlowsheetBlock, ControlVolume0DBlock
+from idaes.core.util import get_solver
+from idaes.config import bin_directory as idaes_bin_directory
+from idaes.generic_models.properties.tests.test_harness import \
+    PropertyTestHarness as PropertyTestHarness_idaes
+from watertap.property_models.tests.property_test_harness import \
+    (PropertyTestHarness, PropertyRegressionTest, PropertyCalculateStateTest)
+
+
+solver = get_solver()
+is_solver_from_idaes_ext = idaes_bin_directory in solver.executable()
+
+eps = 1e-15 # eps value for mass fractions in property package
+
+# -----------------------------------------------------------------------------
+
+class TestNaClProperty_idaes(PropertyTestHarness_idaes):
+    def configure(self):
+        self.prop_pack = props.NaClParameterBlock
+        self.param_args = {}
+        self.prop_args = {}
+        self.has_density_terms = False
+
+class TestDefaultNaClwaterProperty:
+
+    # Create block and stream for running default tests
+    m = ConcreteModel()
+    m.fs = FlowsheetBlock(default={"dynamic": False})
+    m.fs.properties = props.NaClParameterBlock(default=
+                                           {"heat_of_crystallization_model": props.HeatOfCrystallizationModel.constant})
+    m.fs.stream = m.fs.properties.build_state_block([0], default={'defined_state': True})
+
+    m.fs.cv = ControlVolume0DBlock(default={
+        "dynamic": False,
+        "has_holdup": False,
+        "property_package": m.fs.properties})
+    m.fs.cv.add_state_blocks(
+        has_phase_equilibrium=False)
+    m.fs.cv.add_material_balances()
+    m.fs.cv.add_energy_balances()
+    m.fs.cv.add_momentum_balances()
+
+
+    # Create instance of PropertyTesthARNESS class and add attributes needed for tests
+    xv = PropertyTestHarness()
+    xv.solver = 'ipopt-watertap'
+    xv.optarg = {'tol': 1e-8, 'nlp_scaling_method': 'user-scaling'}
```

Comment:
Now that we're using `ipopt-watertap`, In every place we specify `solver` or `optarg` it is not updating the defaults (which is `ipopt-watertap` with `tol=1e-8` and `user-scaling`). Therefore I would recommend just deleting these everywhere.
```suggestion
```

Reply:
Done.